src/PolykeyAgent.ts

```diff
@@ -712,6 +712,25 @@ class PolykeyAgent {
     this.logger.info(`Destroyed ${this.constructor.name}`);
   }
 
+  public async restart({
```

Comment:
Are you using this method anywhere in the tests? This is best separated into another PR.

Reply:
Would include `src/bin/agent/CommandAgent.ts` and `src/bin/agent/CommandRestart.ts`

Reply:
And `src/client/service/agentRestart.ts`.